core/lib/spree/core/engine.rb

```diff
@@ -105,6 +105,12 @@ class Engine < ::Rails::Engine
         ]
       end
 
+      initializer 'spree.promo.register.promotions.shipping_actions', before: :load_config_initializers do |app|
```

Comment:
do we need to keep [`Spree::Promotion::Actions::FreeShipping` into `spree.promo.register.promotions.actions`](https://github.com/jordan-brough/solidus/blob/4db04cac8946c0ca2c041b9302e2f478624d1d93/core/lib/spree/core/engine.rb#L104) as well?